### PR TITLE
qrcode-svg 1.1.0. Completed input options. Organized and documented from package readme.

### PR DESCRIPTION
**Pull request by TobiasMelen**
This updates the definitions with complete input options specified at https://github.com/papnkukn/qrcode-svg#options

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/papnkukn/qrcode-svg#options
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/qrcode-svg/index.d.ts [hunk 1]
```diff
@@ -46,20 +44,41 @@ declare class QRCode {
...
+        /** White space padding. `0` for no border. Default is `4`. */
         padding?: number;
-        /** Default is `256`. */
+        /** QR Code width in pixels. Default is `256`. */
         width?: number;
+        /** QR Code height in pixels. Default is `256`. */
+        height?: number;
+        /** Color of modules (squares), color name or hex string. Default is `#000000`. */
+        color?: string;
+        /** Background - color of background, color name or hex string. Default is `#fffff`. */
```
Comment (ericbf) [inline types/qrcode-svg/index.d.ts]: Can you remove the `Background` prefix in this comment (to be consistent with the other properties)?

File: types/qrcode-svg/index.d.ts [hunk 2]
```diff
@@ -46,20 +44,41 @@ declare class QRCode {
...
+        background?: string;
+        /** Error correction level. Default is `"M"`. */
+        ecl?: 'L' | 'M' | 'H' | 'Q';
+        /** Join modules (squares) into one shape, into the SVG path element, recommended for web and responsive use. Default is `false`. */
```
Comment (ericbf) [inline types/qrcode-svg/index.d.ts]: Can you wrap `recommended` in `**`, to match the docs' emphasis thereof?

File: types/qrcode-svg/index.d.ts [hunk 3]
```diff
@@ -46,20 +44,41 @@ declare class QRCode {
...
+        /** Swap X and Y modules, only if you have issues with some QR readers. Default is `false`. */
+        swap?: boolean;
+        /** Prepend XML declaration to the SVG document, i.e. ```<?xml version="1.0" standalone="yes"?>```. Default is `true`. */
+        xmlDeclaration?: boolean;
+        /**
+         * Wrapping element. Default is `svg`. \
+         * `svg`: Populate squares in a SVG document with `width` and `height` attribute.
+         * Recommended for converting to raster images or PDF where QR Code is being static (exact size). \
+         * `svg-viewbox`: Populate squares in a SVG document with viewBox attriute.
+         * Recommended for responsive web pages. \
```
Comment (ericbf) [inline types/qrcode-svg/index.d.ts]: Can you wrap this `Recommended` in `**`, to match the docs' emphasis thereof?

**Review by ericbf — CHANGES_REQUESTED** [review]
Just a few minor changes in the docs. Looks good otherwise!

**Comment by TobiasMelen** [comment]
Thanks for the review and those catches!
JSDoc comments should be adjusted to feedback.

**Review by ericbf — APPROVED** [review]
Looks good!

**Pull request merged**